docs/source/gbd2019_models/risk_effects/sbp/index.rst

```diff
@@ -303,9 +303,34 @@ Risk Outcome Pair #10: Heart failure
 
 :ref:`See heart failure documentation <2019_cause_heart_failure>`
 
-The relative risks apply to the incidence rates of residual heart failure. 
+In GBD, heart failure is an impairment and does not have a mortality associated with it. For our model, 
```

Comment:
Is there a concern with double-counting mortality here? i.e. does GBD attribute any deaths to IHD that we would consider due to HF?

Reply:
GBD doesn't have any deaths from HF, it is nonfatal only. Deaths are counted under the "cause" instead of under HF, so IHD for example. I think we will just be moving deaths over to a new "cause" and will then undercount for IHD.

Reply:
What, in the sim, will cause us to undercount for IHD? I assumed that we used an IHD excess mortality rate from GBD, which would include excess mortality due to HF. If we then also include excess mortality for HF as a cause, we overestimate all-cause mortality, no?

Reply:
I will need to confirm with Steve, but in the sim IHD is split into myocardial infarction and angina which are modeled with incidence and mortality and so on. I believe that there are mortality rates specific to those instead of the larger IHD cause.

Reply:
We use the following modelable IDs for EMRs (with descriptions taken from the shiny):
* Acute MI: 24694; Myocardial infarction due to ischemic heart disease - EMR Comparison; Running for EMR comparison
* Post-MI; 15755, Post-MI IHD, Model of post-MI IHD prior to exclusivity adjustment with HF and angina
* Stable angina: 1817; Angina due to ischemic heart disease

Not sure if this is helpful but it seems to confirm what you're saying @SylLutze